CakePHP/Sniffs/Functions/FunctionDeclarationArgumentSpacingSniff.php

```diff
@@ -14,17 +14,15 @@
  * @since         CakePHP CodeSniffer 0.1.18
  * @license       http://www.opensource.org/licenses/mit-license.php MIT License
  */
-if (class_exists('Squiz_Sniffs_Functions_FunctionDeclarationArgumentSpacingSniff', true) === false) {
-    $error = 'Class Squiz_Sniffs_Functions_FunctionDeclarationArgumentSpacingSniff not found';
-    throw new PHP_CodeSniffer_Exception($error);
-}
+namespace CakePHP\Sniffs\Functions;
+
+use PHP_CodeSniffer\Standards\Squiz\Sniffs\Functions\FunctionDeclarationArgumentSpacingSniff as FDASSniff;
```

Comment:
Should be FQCN in a use statement
Instead of the unreadable abbr. i would use SquizFunctionDeclarationArgumentSpacingSniff as alias.